lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -436,6 +440,54 @@ def self.get_values(json, category_type, override_price = false)
     # e.g. RDS = http://aws-assets-pricing-prod.s3.amazonaws.com/pricing/rds/mysql/pricing-standard-deployments.js
     @@Compute_Units_Lookup = {}
 
+    # Names here match what are in the aws_instance_type_profiles table
```

Comment:
let's _not_ expose our internal/CHT data table names here (in this public gem)

Reply:
I'm a bit puzzled by the motivation for this, as this "naming" is internal to CHT (and this also makes it public in this gem).  Let's discuss offline.